addons/ondevice-knobs/src/types/Select.js

```diff
@@ -19,7 +19,8 @@ class SelectType extends React.Component {
 
     const options = this.getOptions(knob);
 
-    const selected = options.filter(({ key }) => knob.value === key)[0].label;
+    const active = options.filter(({ key }) => knob.value === key)[0];
```

Comment:
@RWOverdijk Would a find be better here?

Reply:
It would. But so would a for loop, or a POJO instead of an array for lookups. I think in terms of performance it's not a _huge_ deal but I do agree. The downside of find is browser support, I don't know how polyfills work with storybook tbh.